src/event/handlers/DiscordMessageLinkHandler.ts

```diff
@@ -11,10 +11,15 @@ class DiscordMessageLinkHandler extends EventHandler {
 		const messagePreviewService = MessagePreviewService.getInstance();
 		const messageRegex = /https:\/\/(ptb\.)?discord(app)?\.com\/channels\//gm;
 
-		if (message.content.startsWith("!")) return;
-
 		if (message.content.match(messageRegex)) {
 			const linkIndex = message.content.search(messageRegex);
+
+			// Gets the position of the url from a string
```

Comment:
Useless comment, code is self explanatory!

Reply:
Removed 👍